Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'bootstrap', '~> 4.4.1'
 gem 'jquery-rails'
 gem 'omniauth-google-oauth2'
 gem 'figaro'
+gem 'chartkick'
```

Comment:
Oooooooh interesting, I would *definitely* prefer this over dealing with Javascript